fe/fe-core/src/main/java/com/starrocks/common/util/PropertyAnalyzer.java

```diff
@@ -209,10 +206,19 @@ public static Short analyzeReplicationNum(Map<String, String> properties, boolea
             key = PropertyAnalyzer.PROPERTIES_REPLICATION_NUM;
         }
         short replicationNum = Short.parseShort(properties.get(key));
+        checkAvailableBackendsIsEnough(replicationNum);
+        return replicationNum;
+    }
+
+    private static void checkAvailableBackendsIsEnough(short replicationNum) throws AnalysisException {
         if (replicationNum <= 0) {
             throw new AnalysisException("Replication num should larger than 0. (suggested 3)");
         }
-        return replicationNum;
+        List<Long> backendIds = GlobalStateMgr.getCurrentSystemInfo().getAvailableBackendIds();
+        if (replicationNum > backendIds.size()) {
+            throw new AnalysisException("Replication num should less than current alive be node count" + 
```

Comment:
Add the replication num into the error log.
"Replication num should be less than the number of available BE nodes. Replication num is " + replication_num + " available BE nodes is " + backendIds.size();

Reply:
Done